Dockerfile

```diff
@@ -90,7 +90,8 @@ RUN set -x && \
     mkdir -p /usr/local/share/package-licenses && \
     hack/copy-modules-license.sh /usr/local/share/package-licenses ./cmd/pmem-csi-driver ./cmd/pmem-vgm ./cmd/pmem-ns-init && \
     cp /go/LICENSE /usr/local/share/package-licenses/go.LICENSE && \
-    cp LICENSE /usr/local/share/package-licenses/PMEM-CSI.LICENSE
+    cp LICENSE /usr/local/share/package-licenses/PMEM-CSI.LICENSE && \
+    if [ "$BIN_SUFFIX" = "-test" ]; then GOOS=linux GO111MODULE=on go build -o _output/pmem-dax-check ./test/cmd/pmem-dax-check && mv _output/pmem-dax-check /usr/local/bin/pmem-dax-check && swupd bundle-add fio; fi
```

Comment:
I think that belongs after `mv _output/pmem-ns-init${BIN_SUFFIX} /usr/local/bin/pmem-ns-init` - that'll make the diff smaller, too.

Please break into multiple lines.

You can remove one command with `go build -o /usr/local/bin/pmem-dax-check`.